src/js/_store/modules/categories.js

```diff
@@ -0,0 +1,32 @@
+import axios from '@Helpers/axiosDefault';
+
+export default {
+  namespaced: true,
+  state: {
+    categoryList: [],
+    errorMessage: '',
+  },
+  mutations: {
+    failRequest(state, { message }) {
+      state.errorMessage = message;
+    },
+    doneGetAllCategories(state, payload) {
+      state.categoryList = payload.categories;
+    },
+  },
+  actions: {
+    getAllCategories({ commit, rootGetters }) {
+      axios(rootGetters['auth/token'])({
+        method: 'GET',
+        url: '/category',
+      }).then(res => {
+        const payload = {
+          categories: res.data.categories.reverse(),
+        };
+        commit('doneGetAllCategories', payload);
```

Comment:
わざわざ`payload`にobjectで格納しなくても良いのではないかと思います！

例えば、
```suggestion
      }).then(res => {
        commit('doneGetAllCategories', res.data.categories.reverse());
```
もしくは、
```suggestion
      }).then(res => {
        const categoryListData = res.data.categories.reverse();
        commit('doneGetAllCategories', categoryListData);
```
というようにしてあげると、`doneGetAllCategories`で代入するものはpayloadだけ、とシンプルになるのでは無いかと思いました！

Reply:
ありがとうございます。
1つ目の修正案のやり方で修正しました。

95e5318e1b6af8360a58eee11ed4e45f06656396

APIから受け取る配列は一つだけなので、わざわざpayloadに入れる必要はないんですね。